nym-vpn-lib/src/platform/android.rs

```diff
@@ -215,6 +305,9 @@ enum Error {
     #[error(display = "Failed to select() on tunnel device")]
     Select(#[error(source)] nix::Error),
 
+    #[error(display = "Gateway does not contain a two character country ISO")]
```

Comment:
Just a side note for something to consider in the future: convention in is to start Display impl strings for error types with lowercase, since usually they are embedded as part of another string at the top-level when they are display. E.g
```
if let Err(err) = foo() {
    error!("Failed to call foo: {err}");
}
```